### ISSUE TITLE
BASIC example CONT, example gives an "?can't continue error"

### ISSUE THREAD

**Issue by MauriceMEGA65** [MEGA65/mega65-user-guide#211]
Is the error supposed to happen? If yes, we should include it in the screen example

**Comment by gurcei** [comment]
The "?cant continue error" shouldn't happen, it should successfully continue. So it appears that "CONT" is broken in BASIC65.

But to be fair, I also tested it with the original 911001.BIN rom, and it is also broken there too.

It does however work in C64-mode.

I'll have a little browse of the rom-source to see if it's something easy to fix, but it it's beyond my abilities, we might have to defer the fix for later.

**Comment by gurcei** [comment]
Ok, from my debugging, I'm seeing that breaking a program with RUN-STOP will cause **"b65.src" : errisd() --> init_stack()** to trigger:

```
**********
init_stack
**********
...
    sta subflg      ;allow subscripted & integer vars
    sta oldtxt+1    ;disallow continuing
    rts
```
So it looks like there's some intention in that comment to disable continuing.

As a test, I tried commenting this line out, and yes, the CONT command works once more, but I feel hesitant committing in such a change without assuring it won't break anything else :D 

It'd be nice if we could get some automated tests running for such changes, Ubik made some efforts in this direction, I should try and get more familiar with it.

**Comment by gurcei** [comment]
I ran Ubik's test suite, that all passed, but we've got other tests that have accumulated in our repos, it'd be nice to get those to run as part of some automated process to build more confidence in any small changes like this.

One of the reasons for my worry is that for an original developer to do this, and comment their intention to "disallow continuing", it makes me wonder the reasoning behind this intent. Was there an issue with continuing on certain commands that would cause other things to break, so they were mulling whether to disable/abandon the continue command, so as to avoid the issue?

Anyway, I'm thinking I'll take Bit Shifter's approach on this, and just let the document reflect what we **aim** for the behaviour to be, and sort out the resolution of the problem at a later time (after doc-printing preparations are sorted out).

Happy to leave this card open, to remind us to get around to fixing this eventually.
